### PR TITLE
Add basic Docusaurus structure

### PR DESCRIPTION
**Pull request by oscarpolanco**
Adding basic docs structure to the repo.

# Steps for testing the site locally:
- Go to the `website` folder
- Install all dependencies using `npm install`
- Start your local server using `npm run start`
- Test locally at [localhost](http://localhost:3000/)

# Steps to test the build:
- Go to the `website` folder
- Install all dependencies using `npm install`
- Start your local server using `npm run build`
- Check that now you got a `build` folder under the `website` directory

# Steps to test the doc versioning:
- Go to the `website` folder
- Install all dependencies using `npm install`
- Start your local server using `npm run version 0.0.0` (number of your version)
- Start your local server using `npm run start`
- Go to [localhost](http://localhost:3000/)
- On the navbar you should see the number of your new doc version
- Go to [version page](http://localhost:3000/amphora/versions)
- Check if your version is on the list

# First deploy process (manual):
- Create a local branch call `gh-pages` from `master`
- Go to the `website` folder
- Install packages using `npm install`
- Do a build using `npm build`
- go back to the `root` folder
- Add the files and commit using this command => `git add -f website/build/Amphora/ && git commit -m"deploy files"`
- Push the static files using this command => `git  subtree push --prefix website/build/Amphora origin gh-pages`
## Note:
- This Process I will do it when the` PR` is approved
- We only need the static files on the `gh-pages` branch and the commit history only will show docs update and important remember that the static files are part of the `.gitignore` that's why we add and push this way.
- Any suggestions to have a better first deploy process are more than welcome 😄 

# General Notes: 
- There are some duplicate files until we got the live docs page
- We still don't have the final design
- Leave some of the default files for future references

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+Assuming you have configured your sites properly and have some components, you'll now be able to access any Clay core routes and any data included in [internal bootstrap files](bootstrap.md) for components and sites will have been added.
+
+## Instantiation Arguments
+
+At instantiation time Amphora accepts a config object which contains properties that affect everything from user sessions to plugins for enhanced functionality. Below is a quick list of arguments with references to further documentation resources on the subjects.
+
+* `app`: an Express instance. If one is not passed in, Amphora will create one during startup. The advantage passing an instance into Amphora is the ability to add any middleware or other configuration necessary for your specific setup.
+* `providers`: an Array of strings which correspond to the supported authentication services in Amphora: `apikey`, `ldap`, `google`, `slack`, and `twitter`. For further information on configuring these providers see the [Authentication](https://github.com/clay/amphora/tree/3a300d4ec7af113afd102b4506e7566eb617c9c8/docs/lifecycle/startup/authentication.md) page.
```
Comment (oscarpolanco) [inline docs/install.md]: Missing link here

File: docs/install.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+* `sessionStore`: used for session management with user sessions when a provider is configured. For more information see the [Authentication](https://github.com/clay/amphora/tree/3a300d4ec7af113afd102b4506e7566eb617c9c8/docs/lifecycle/startup/authentication.md) page.
```
Comment (oscarpolanco) [inline docs/install.md]: Missing link here

File: docs/install.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+* `renderers`: an Object that references modules that can be used to transform component data into different formats. For example, [Amphora HTML](https://github.com/clay/amphora-html) is a module that renders component data to HTML using Handlebars templates. Renderers abide by an API that Amphora sets forth. The `renderers` Object should contain properties whose names correspond to request extensions and whose properties are the handlers for those extensions. A `default` property is used to specify the renderer to be used when no extension is specified in a request. For more information, see the [Renderers](https://github.com/clay/amphora/tree/3a300d4ec7af113afd102b4506e7566eb617c9c8/docs/lifecycle/startup/authentication.md) documentation.
+* `plugins`: an Array of Objects that have handlers for different plugin hooks that Amphora exposes. Different hooks are exposed for the startup, request and publish life cycles. For more information see the [Plugins](https://github.com/clay/amphora/tree/3a300d4ec7af113afd102b4506e7566eb617c9c8/docs/lifecycle/startup/plugins.md) page.
```
Comment (oscarpolanco) [inline docs/install.md]: Missing link here

File: website/README.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+        └── logo.svg
+    └── versioned_docs
+      └── version 0.0.0
+        ├── docs.md
+    └── versioned_sidebars
+      ├── version-0.0.0-sidebars.json
+```
+
+# Publish the website
+The Amphira website will live on GitHub page. You just need to run the following commands:
```
Comment (manuelurenah) [inline website/README.md]: Typo in here
```suggestion
The Amphora website will live on GitHub page. You just need to run the following commands:
```

File: website/siteConfig.js [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+  // Show documentation's last contributor's name.
+  // enableUpdateBy: true,
+
+  // Show documentation's last update time.
+  // enableUpdateTime: true,
+
+  // You may provide arbitrary config keys to be used as needed by your
+  // template. For example, if you need your repo's URL...
+  //   repoUrl: 'https://github.com/facebook/test-site',
```
Comment (manuelurenah) [inline website/siteConfig.js]: Do you think it is possible to remove the commented code? Or do you think is a better idea to leave it as it is?

Comment (oscarpolanco) [reply website/siteConfig.js]: We can delete it. We will have a clean config file

File: .istanbul.yml [hunk 6]
```diff
@@ -2,7 +2,7 @@ verbose: false
 instrumentation:
     root: .
     default-excludes: true
-    excludes: []
+    excludes: ['**/website/**']
```
Comment (fxisco) [inline .istanbul.yml]: why we exclude this?

File: website/core/Footer.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const React = require('react');
+
+class Footer extends React.Component {
```
Comment (fxisco) [inline website/core/Footer.js]: Are this are the same for all the projects?

File: .istanbul.yml (hunk 6 above)
Comment (oscarpolanco) [reply .istanbul.yml]: We don't handle `jsx` when we run the test. For now, we skip the test on the docs.

File: website/core/Footer.js (hunk 7 above)
Comment (oscarpolanco) [reply website/core/Footer.js]: Yes
These files are generated by docusaurus

**Pull request merged**